src/osd/scheduler/mClockScheduler.h

```diff
@@ -186,15 +206,32 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
   // Enqueue op in the back of the regular queue
   void enqueue(OpSchedulerItem &&item) final;
 
-  // Enqueue the op in the front of the regular queue
+  // Enqueue the op in the front of the high priority queue or the immediate queue (based on priority)
   void enqueue_front(OpSchedulerItem &&item) final;
 
+  // Enqueue the op to the high priority queue
+  void enqueue_strict(unsigned prio, OpSchedulerItem &&item, bool front = false);
+
   // Return an op to be dispatch
   WorkItem dequeue() final;
 
```

Comment:
Because high_priority items are never empty, this is unnecessary, simply use high_priority.empty().